### PR TITLE
headless ui test

### PR DESCRIPTION
**Pull request by manrodrigues**
- enable end-to-end tests to run in headless mode

### CODE REVIEW

File: python-test/features/steps/control_agents_ui.py [hunk 1]
```diff
@@ -92,9 +92,6 @@ def create_agent_through_the_agents_page(context, orb_tags):
 @then("the agents list and the agents view should display agent's status as {status} within {time_to_wait} seconds")
 def check_status_on_orb_ui(context, status, time_to_wait):
     context.driver.get(f"{orb_url}/pages/fleet/agents")
-    # agent_xpath = f"//span[contains(text(), '{context.agent_name}')]/ancestor::div[contains(@class, " \
```
Comment (etaques) [inline python-test/features/steps/control_agents_ui.py:95]: This comment is right?

File: python-test/test_config.ini.tpl [hunk 2]
```diff
@@ -16,3 +16,4 @@ This field is required if use docker approach to run the tests
 # orb_agent_interface=mock
```
Comment (etaques) [inline python-test/test_config.ini.tpl:16]: Could you un-comment this lines to template be ready to use on pipeline?

**Review by etaques — CHANGES_REQUESTED** [review]
Small changes to be ready on pipeline use

File: python-test/features/steps/control_agents_ui.py (hunk 1 above)
Comment (manrodrigues) [reply python-test/features/steps/control_agents_ui.py:95]: It's already removed :)

**Review by etaques — APPROVED** [review]
LGTM

**Pull request merged**